Dockerfile.pulumi

```diff
@@ -4,8 +4,8 @@ FROM python:${PYTHON_VERSION}-slim-bullseye as grapl-local-pulumi
 RUN apt-get update \
     && apt-get install --yes --no-install-recommends \
        build-essential=12.9 \
-       curl=7.74.0-1.3+deb11u2 \
-       unzip=6.0-26 \
+       curl=7.74.0-1.3+deb11u3 \
+       unzip=6.0-26+deb11u1 \
     && rm -rf /var/lib/apt/lists/* \
     && adduser \
```

Comment:
I assume these are covered in another pr?

Reply:
I don't even understand how that's in here. I did a rebase from main and skipped the cherrypicked commits. wtf